README.md

```diff
@@ -161,9 +161,9 @@ transcript = transcript_list.find_generated_transcript(['de', 'en'])
 ```
 
 ### Using Formatters
-Formatters are meant to be an additional layer of processing of the transcript you pass it. The goal is to convert the transcript from its Python data type into a consistent string of a given "format". Such as a basic text (`.txt`) or even formats that have a defined specification such as JSON (`.json`), WebVTT format (`.vtt`), Comma-separated format (`.csv`), etc...
+Formatters are meant to be an additional layer of processing of the transcript before you pass it. The goal is to convert the transcript from its Python data type into a consistent string of a given "format". Such as a basic text (`.txt`) or even formats that have a defined specification such as JSON (`.json`), WebVTT format (`.vtt`), Comma-separated format (`.csv`), etc...

```

Comment:
I would say it did make more sense with out the `before`, as the processing happens after passing the transcript in, not before.